vscode/src/projectSystem.ts

```diff
@@ -247,6 +248,11 @@ export async function fetchGithubRaw(
   const uri = `${githubEndpoint}/${owner}/${repo}/${ref}/${pathNoLeadingSlash}`;
   log.info(`making request to ${uri}`);
   const response = await fetch(uri);
```

Comment:
Network errors (including CORS) cause the `await fetch` to throw, in which case it will never get to the sendTelemetryEvent line.

That said, we only support GitHub (which should never see a CORS error), and if there are network issues telemetry probably wont send either, so may be fine. Maybe worth a comment to note for future readers though.